.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           POSTGRES_DB: jetdb
           PGPORT: 50901
 
-      - image: circleci/mysql:8.0.27
+      - image: cimg/mysql:8.0
```

Comment:
Why is it necessary?

Reply:
I'll revert the version change, I wasn't able to pull mysql:8.0.27, I assumed that was the case for circleci.  

Regarding the namespace, https://circleci.com/docs/circleci-images/

```
Legacy images with the prefix “circleci/” were [deprecated](https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034)
 on December 31, 2021. For faster builds, upgrade your projects with [next-generation convenience images](https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic).
 ```

Reply:
Actually, this may still be valid.  The CI and tests (docker-compose) can't point to the same version of MySQL since mysql:8.0.27 is no longer available while {cimg,circleci}/mysql:8.0.27 does exist.  Would you prefer both to be on 8.0 or leave the discrepancy?

Reply:
If you revert `.circleci/config.yml` and `docker-compose.yaml` the test should pass. `cimg/mysql:8.0` will download latest mysql version(v39) and tests are catching some differences in that version.

Reply:
I had no problem pulling `circleci/mysql:8.0.27`.

Reply:
I reverted though keep in mind `docker pull mysql:8.0.27` from docker-compose will fail.